webprojects/SUMO.md

```diff
@@ -0,0 +1,66 @@
+# Support.mozilla.org(SUMO): how to localize and test
+
+SUMO, short for **SU**pport.**MO**zilla.org, is a knowledge base site, providing support to Mozilla product users. More than half of all Firefox users speak a language other than English. Mozilla depends on contributors to make support available to people around the globe.
+
+SUMO consists of two parts: the product that contains strings in user interfaces or UI, such as navigations and buttons; and the support articles that are either for troubleshoot or the know-hows for every Mozilla product. This document covers the product side where localization takes place in Pontoon.
+
+## Localizing the strings
+
+There are two active components in Pontoon that make up the [SUMO product](https://pontoon.mozilla.org/sumo):
+
+* **LC_MESSAGES/djangojs.po** contains strings for the JS bits of the platform.
+* **LC_MESSAGES/django.po** contains the majority of strings in the SUMO product.
+
+SUMO product is made up by several file types. In order to minimize revision and get translation right the first time, the key is to get the context right.  Look for clues or ask others for help:
+* [String ID](https://pontoon.mozilla.org/zh-TW/sumo/LC_MESSAGES/django.po/?string=150684) shows where the strings is used.
```

Comment:
This doesn't make sense, you're pointing at a comment listing the files where this string is used ;-)